internal/core/src/segcore/SegmentInterface.cpp

```diff
@@ -293,4 +293,19 @@ SegmentInternalInterface::timestamp_filter(BitsetType& bitset,
     }
 }
```

Comment:
at present, SkipIndex is a segment-level component, and relating load is also triggered by segment